composer.json

```diff
@@ -10,7 +10,7 @@
     ],
     "require": {
         "php": ">=5.5.0",
-        "guzzlehttp/guzzle": "~5.0"
```

Comment:
I think it is possible to be compatible with guzzle ~5.3 and ~6.0 at the same time. Laravel, and aws do that.